build.sbt

```diff
@@ -15,7 +15,7 @@ lazy val root = (project in file("."))
   .enablePlugins(PlayScala, PlayAkkaHttpServer)
   .disablePlugins(PlayNettyServer)
 
-val AwsVersion = "2.18.27"
+val AwsVersion = "2.21.2"
```

Comment:
Fix: Upgrade to software.amazon.awssdk:cloudwatch@2.21.10 
I think 2.21.2 still have high vuln.

Reply:
Cannot see a high vuln in the [snapshot](https://app.snyk.io/org/guardian-capi/project/6317798d-c26e-456a-84dc-8df25c949d53/history/482c74b6-d377-4b63-9aa5-837b3fc74400) @Divs-B, but if I've missed something let me know?

Reply:
Yeah high snyk vuln seems resolved with 2.21.2 version (just tested locally on your branch).

It will be then nicer to have 2.21.10 as recommended, 
but can do in future PR, Thanks @jonathonherbert. good to go 👍

Reply:
Oh I see what you mean, 2.21.10 includes the fix – thanks for spotting! 54810a78af1b7329e3d3fdb4a297536091b0ce20 makes the change.